src/app/shared/search/advanced-search/advanced-search.component.scss

```diff
@@ -0,0 +1,38 @@
+:host .facet-filter {
```

Comment:
@GauravD2t 

this seems to be the same scss of the [search-filter.component.scss](https://github.com/DSpace/dspace-angular/blob/c7197826a8e48f795e59cb6f98fb17c5f11983a7/src/app/shared/search/search-filters/search-filter/search-filter.component.scss)  so maybe it's better to use the same file within this component instead of duplicate it